packages/redux-simple-auth/src/createAuthenticator.js

```diff
@@ -1,10 +1,9 @@
-export default (
-  {
-    name,
-    restore = () => Promise.reject(),
-    authenticate = () => Promise.reject()
-  } = {}
-) => {
+export default ({
+  name,
+  restore = () => Promise.reject(),
+  authenticate = () => Promise.reject(),
+  invalidate = () => Promise.resolve()
```

Comment:
Since the default behavior is to resolve here, could we resolve with the `data` argument passed to this?